umbraco/Umbraco-CMS
# Don't obsolete the v2 pickers, renamed them and they're able to be created again, without config changes

Fixes #10423

To test: 
- Do a fresh install and notice the media picker is there named `Media Picker (legacy)` - if you install the starter kit, this is what pickers are use
- Upgrade an 8.13 site, notice that the picker is still names `Media Picker` without `(legacy)` since we don't do any renaming during upgrades
- In either install, notice that the `legacy` picker can still be created, without config changes
